src/GovUk.Education.ExploreEducationStatistics.Common/Model/FileInfo.cs

```diff
@@ -12,6 +12,8 @@ public class FileInfo
         public string FileName { get; set; }
         public string Name { get; set; }
         public string Size { get; set; }
+        public string UserName { get; set; }
```

Comment:
This class is used as a view model despite not containing the name `ViewModel`.
It lives in the `Common` package because it's being used in several places including the Public by the Publisher and the Content API.

With this change these fields might be persisted to the cached public JSON content and possibly returned in the public content API JSON response too, depending on the serialization config and whether its set to include null fields in JSON or omit them.

We might want a public file view model that doesn't contain the `UserName` fields and probably the `Created` field too ruling out the possibility of us ever exposing them.